### PR TITLE
feat(@angular-ru/common): add NgCssClasses

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by Schirbak — APPROVED** [review]

File: packages/common/docs/typings.md [hunk 1]
```diff
@@ -162,3 +162,9 @@ interface SomeInterface {
 type SomeStringNumberKeys = KeysOfType<SomeInterface, string | number>;
 // SomeStringNumberKeys === 'someNumber' | 'someString' | 'someAnotherNumber'
 ```
+
+-   `NgCssClasses`
+
+```ts
+const myClasses = { 'element__block--modificator': true };
```
Comment (nikitasadchikov) [inline packages/common/docs/typings.md:169]: `block__element--modificator` )

**Review by nikitasadchikov — CHANGES_REQUESTED** [review]

File: packages/common/typings/src/ng-css-classes.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { Any } from './any';
+
+export type NgCssClasses = string | string[] | Set<string> | { [klass: string]: Any };
```
Comment (mansurishmuhametov) [inline packages/common/typings/src/ng-css-classes.ts:3]: klass ?

File: packages/common/docs/typings.md (hunk 1 above)
Comment (splincode) [reply packages/common/docs/typings.md:169]: thank you ))

File: packages/common/typings/src/ng-css-classes.ts (hunk 2 above)
Comment (nikitasadchikov) [reply packages/common/typings/src/ng-css-classes.ts:3]: it's ok due to 'class' is reserved

**Review by nikitasadchikov — APPROVED** [review]

**Review by mansurishmuhametov — APPROVED** [review]

**Pull request merged**